tsconfig.shared.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "./tsconfig.root.json",
+  "compilerOptions": {
+    "declaration": true,
+    "declarationMap": true,
+    "emitDeclarationOnly": true
+  }
+}
```

Comment:
This is common setting for `*-shared` workspaces. Their compilation will be done by `swc`, therefore `tsc`'s transpilation must be turned off.